tools/depends/target/samba-gplv3/Makefile

```diff
@@ -56,12 +56,21 @@ endif
 
 $(LIBDYLIB): $(PLATFORM)
 	$(MAKE) -C $(PLATFORM)/source3 libsmbclient
+ifeq (darwin, $(findstring darwin, $(HOST)))
+	$(MAKE) -C $(PLATFORM)/source3 libtevent
+endif
 
 .installed-$(PLATFORM): $(LIBDYLIB)
 	$(MAKE) -C $(PLATFORM)/source3 installlibsmbclient
 ifeq (darwin, $(findstring darwin, $(HOST)))
+	$(MAKE) -C $(PLATFORM)/source3 installlibtevent
 	mv $(PREFIX)/lib/libsmbclient.dylib.0 $(PREFIX)/lib/libsmbclient.dylib
+	mv $(PREFIX)/lib/libtevent.dylib.0      $(PREFIX)/lib/libtevent.dylib
+	mv $(PREFIX)/lib/libtevent.dylib.0.9.11 $(PREFIX)/lib/libtevent.dylib
```

Comment:
also this looks a bit strange - remove one and move the other would look a bit cleaner